### ISSUE TITLE
[4.0] Could not create new article on the fly via a modal from the menu item

### ISSUE THREAD

**Issue by uglyeoin** [joomla/joomla-cms#24628]
### Steps to reproduce the issue
Create a new menu item "single article"

Click to create the article from within the menu item.

Fill in the category, title and some content

### Expected result

Am able to save or save and close

### Actual result

Am able to close only but not save.  It gives no feedback.  

I believe it is a modal issue as I see this in the console.

```
modal-fields.min.js?16074c9d9515d193ebbba671d7c573c4:1 Uncaught TypeError: Cannot read property 'formvalidator' of null
    at window.processModalEdit (modal-fields.min.js?16074c9d9515d193ebbba671d7c573c4:1)
    at HTMLAnchorElement.onclick (index.php?option=com_menus&view=item&client_id=0&menutype=mainmenu&layout=edit:301)
window.processModalEdit @ modal-fields.min.js?16074c9d9515d193ebbba671d7c573c4:1
onclick @ index.php?option=com_menus&view=item&client_id=0&menutype=mainmenu&layout=edit:301
```

### System information (as much as possible)

[systeminfo-2019-04-17T11_09_01-04_00.txt](https://github.com/joomla/joomla-cms/files/3090275/systeminfo-2019-04-17T11_09_01-04_00.txt)

**Comment by ghost** [comment]
@uglyeoin Please add "[4.0] " at front of Title for J4 Issues.

**Comment by uglyeoin** [comment]
Awesome thanks, I was wondering why that didn't happen, didn't realise I should be doing it.

**Comment by ghost** [comment]
@uglyeoin thanks :-) This is status quo (Details #24509).

**Comment by Arpit-24** [comment]
Will get fixed by merging PR #24350

**Comment by Quy** [comment]
Please test PR #24350<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/24628">issues.joomla.org/tracker/joomla-cms/24628</a>.</sub>
